debian/not-installed

```diff
@@ -0,0 +1 @@
+usr/lib/*/*.la
```

Comment:
This avoids a `dh_missing(1)` warning due to this file being generated but not installed.